### PR TITLE
Payload deployment, ROOT 6.30

### PR DESCRIPTION
**Pull request by harrypuuter**
This PR adds several small changes

- Update to LCG 105 using ROOT 6.30
- Add analysis_configuration/$ANALYSIS/payloads folder to the installdir, if available. Within the CROWN config, files from the payloads folder can be accessed using `payloads/path/to/file`. (@nfaltermann )
- Fixed a bug, where the CROWNLIB was not requild, requiring a removal of the libfile (@conformist89 )

### CODE REVIEW

**Comment by harrypuuter** [comment]
pls let me know, if anything breaks when using ROOT 6.30

File: CMakeLists.txt [hunk 1]
```diff
@@ -339,6 +353,9 @@ STRING(REGEX REPLACE "\n" ";" FILELIST ${FILELIST})
 set(TARGET_NAMES "")
 # copy all correction files into the install location
 install(DIRECTORY data/ DESTINATION ${INSTALLDIR}/data)
+if (PAYLOADS)
+    install(DIRECTORY ${CMAKE_SOURCE_DIR}/analysis_configurations/${ANALYSIS}/payloads DESTINATION ${INSTALLDIR}/payloads)
```
Comment (nfaltermann) [inline CMakeLists.txt:357]: I got `build/bin/payloads/payloads/` on a test, I guess it's just `DESTINATION ${INSTALLDIR}/`?

Comment (harrypuuter) [reply CMakeLists.txt]: True, should be fixed now

Comment (nfaltermann) [reply CMakeLists.txt]: thanks, works fine

also, progress bar goes brrrrrrr

**Comment by conformist89** [comment]
It works good to me, compatible with ROOT 6.30 and crownlib bug seems to be corrected now

**Review by conformist89 — APPROVED** [review]

**Pull request merged**